### PR TITLE
[bitnami/thanos] introduce logFormat in Thanos components

### PR DESCRIPTION
**Pull request by jleloup**
**Description of the change**

Current Thanos chart does not allow users to set log format: the default `logfmt` format is implicitly used by Thanos.

This Pull Request introduce a set of Helm values to do so in each Thanos component: using `logfmt` as default to prevent any change in behaviour for users that don't want to change this format.

**Benefits**

This change allows users to set a different format for Thanos logs: especially JSON format comes to mind.

**Possible drawbacks**

No drawback identified.

**Applicable issues**

No issue currently opened related to this change.

**Additional information**

**Checklist** 

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/thanos/templates/ruler/statefulset.yaml [hunk 1]
```diff
@@ -86,6 +86,7 @@ spec:
           args:
             - rule
             - --log.level={{ .Values.ruler.logLevel }}
+            - --log.format={{ .Values.bucketweb.logFormat }}
```
Comment (rafariossaa) [inline bitnami/thanos/templates/ruler/statefulset.yaml:89]: I think there is a copy& paste error here, shouldn't this be `.Values.ruler.logFormat` ?

If it is, please review the other places where `logFormat` is used.

**Review by rafariossaa — CHANGES_REQUESTED** [review]
Hi,
  Thank you very much for your PR !.

  I think I have found an issue, could you take a look to it ?

File: bitnami/thanos/templates/ruler/statefulset.yaml (hunk 1 above)
Comment (jleloup) [reply bitnami/thanos/templates/ruler/statefulset.yaml:89]: Indeed, I'm caught red handed :)
I'll double check other values.

Comment (jleloup) [reply bitnami/thanos/templates/ruler/statefulset.yaml]: Fixed

**Review by rafariossaa — APPROVED** [review]
Thanks !
This LGTM

**Pull request merged**